raiden/tests/utils/mock_client.py

```diff
@@ -421,9 +418,9 @@ def settled(self):
         return self.contract.settled
 
     def detail(self, our_address):
-        partner_address = self.contract.partner(our_address)
-
         our_balance = self.contract.participants[our_address].deposit
+
+        partner_address = self.contract.partner(our_address)
```

Comment:
Since you're removing the `partner` functions in another PR, you should probably update here to use the `addressAndBalance` function.